scene/resources/text_paragraph.cpp

```diff
@@ -72,6 +72,11 @@ void TextParagraph::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_flags", "flags"), &TextParagraph::set_flags);
 	ClassDB::bind_method(D_METHOD("get_flags"), &TextParagraph::get_flags);
 
+	ClassDB::bind_method(D_METHOD("set_text_overrun_behavior", "overrun_behavior"), &TextParagraph::set_text_overrun_behavior);
+	ClassDB::bind_method(D_METHOD("get_text_overrun_behavior"), &TextParagraph::get_text_overrun_behavior);
+
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "text_overrun_behavior", PROPERTY_HINT_ENUM, "Trim nothing,Trim characters,Trim words,Ellipsis,Word ellipsis"), "set_text_overrun_behavior", "get_text_overrun_behavior");
```

Comment:
You're separating the `flags` property from its method with this addition here, it's inconsistent with existing code.

Reply:
Oops, fixed.